torch/_inductor/utils.py

```diff
@@ -700,9 +700,10 @@ def patched_compile_to_module(self):
 
 def run_and_get_triton_code(fn, *args, **kwargs):
     _, source_codes = run_and_get_code(fn, *args, **kwargs)
+    # Can have two outputs if backwards was eagerly compiled
```

Comment:
Can we store a flag to drive if this should be exactly 1, or of (1, 2)?

Reply:
It's very awkward, because the current implementation will attempt to compile backwards, and if backwards failed to compile, suppress the error and return anyway. So there isn't really a clear delineation.